tree/30_generic_methods/_file_check_special.cf

```diff
@@ -23,7 +23,7 @@
 # @parameter file_name File name
 # @parameter file_type File type to check, can be "block device", "character device", "directory", "FIFO/pipe", "regular file", "socket", "symlink"
 # 
-# @class_prefix file_check_special
+# @class_prefix file_check
```

Comment:
neither of these if actually right, but it is not a real generic method so that's ok